Userland/Libraries/LibURL/URL.h

```diff
@@ -58,6 +58,11 @@ enum class ApplyPercentDecoding {
     No
 };
 
+struct BlobURLEntry {
+    String type;
+    ByteBuffer byte_buffer;
+};
```

Comment:
Yeah, this is the part I'm not super happy with since it's pretty naive. And we still need to shuffle stuff around (such as put BlobURLRegistry in a singleton process like webserver) even if we put a workers local blob for it's worker here. I'll leave a FIXME though, it's definitely awkward